### PR TITLE
BAQE-2176 - Archive all logs in case of an error

### PR DESCRIPTION
**Pull request by MarianMacik**
@jiripetrlik Here's the fix for always archiving logs.

### CODE REVIEW

File: kogito-benchmarks-framework/src/main/java/org/kie/kogito/benchmarks/framework/Logs.java [hunk 1]
```diff
@@ -172,8 +173,15 @@ public static void archiveLog(String testClass, String testMethod, File log) thr
...
         Path destDir = getLogsDir(testClass, testMethod);
         Files.createDirectories(destDir);
-        String filename = log.getName();
-        Files.copy(log.toPath(), destDir.resolve(filename), REPLACE_EXISTING);
+        try (Stream<Path> walk = Files.walk(appLogsDir)) {
+            walk.skip(1).forEach(path -> { // skip root directory
+                try {
+                    Files.copy(path, destDir.resolve(appLogsDir.relativize(path)), REPLACE_EXISTING);
+                } catch (IOException e) {
+                    e.printStackTrace();
```
Comment (jstastny-cz) [inline kogito-benchmarks-framework/src/main/java/org/kie/kogito/benchmarks/framework/Logs.java:181]: is this print instead of log on purpose?

Comment (MarianMacik) [reply kogito-benchmarks-framework/src/main/java/org/kie/kogito/benchmarks/framework/Logs.java]: You are right, I thought I would change it when I did it, but forgot. Now it's fixed.

**Review by jstastny-cz — APPROVED** [review]
LGTM.

**Review by jiripetrlik — APPROVED** [review]
Thank you. Looks good to me.

**Pull request merged**